### PR TITLE
[Fix] Fixed vehicle info not fitting in the info box

### LINKED ISSUE #1068: Vehicle Info not showing completly
Hello again...

This issue is actually there since a few Night Build.

I want you developers to make this issue a priority.
image

There's a few information such as damage %.

### PR DESCRIPTION
**Pull request by AnotherFoxGuy**
Fixes #1068

screenshot_2016-07-14_12-47-06_2

### CODE REVIEW

File: tools/linux/l10n/from_transifex.sh [hunk 1]
```diff
@@ -13,4 +13,6 @@ do
 	echo "Now copying $file"
 	mkdir -p ../../../languages/$file/LC_MESSAGES/
 	cp $file.po ../../../languages/$file/LC_MESSAGES/ror.po
+	cp ../../../SourceSansPro-Regular.ttf ../../../languages/$file/LC_MESSAGES/SourceSansPro-Regular.ttf
```
Comment (Hiradur) [inline tools/linux/l10n/from_transifex.sh]: This seems inefficient to me, can't you just put it in the `resources` folder?

File: tools/linux/l10n/SourceSansPro-License.txt [hunk 2]
```diff
@@ -0,0 +1,91 @@
+Copyright (c) 2010, Adobe Systems, Inc. (http://www.adobe.com/)
```
Comment (Hiradur) [inline tools/linux/l10n/SourceSansPro-License.txt]: This text should be appended to DEPENDENCIES.md

**Comment by Hiradur** [comment]
It's good but  having to copy the MyGUI_FontConfig.xml file to every language folder still seems to be inefficient. Why aren't we using SourceSansPro-Regular.ttf fo English as well so one MyGUI_FontConfig.xml file in the bin/resources/mygui folder should be enough?
One issue with SourceSansPro is that it's rather limited. As far as I can tell it doesn't have e.g. kyrillic, arabic or asian characters. IMO GNU Unifont would be a good choice as a "one for all" font. Also see #450

**Comment by ulteq** [comment]
@AnotherFoxGuy, can you comment on what @Hiradur said?

**Comment by AnotherFoxGuy** [comment]
>Why aren't we using SourceSansPro-Regular.ttf fo English as well so one MyGUI_FontConfig.xml file in the bin/resources/mygui folder should be enough?

I've moved SourceSansPro-Regular.ttf and MyGUI_FontConfig.xml to the resources folder.

**Comment by ulteq** [comment]
> Edit: it seems that Travis is broken?

Nah, it works now.

**Pull request merged**